packages/jsii-python-runtime/src/jsii/_kernel/providers/process.py

```diff
@@ -244,7 +256,9 @@ def _jsii_runtime(self):
         return paths[0]
 
     def _next_message(self) -> Mapping[Any, Any]:
-        return json.loads(self._process.stdout.readline(), object_hook=ohook)
+        # return json.loads(self._process.stdout.readline(), object_hook=ohook)
```

Comment:
Probably need to just delete the line of code here? Or Uncomment it and delete the next two lines.